Components/Basic/custom.component.yaml.meta.yaml

```diff
@@ -1,7 +1,33 @@
 ---
 docstring: |
     ## custom.component.yaml
-    *TODO*
+    Use a custom Python component.
```

Comment:
Warrants a better description.

example/ Custom components are where you can call out to your backend systems, do more complex data processing, etc.